service/provider/src/main/java/gov/va/vro/service/provider/bgs/service/BgsClaimNotes.java

```diff
@@ -1,13 +1,13 @@
 package gov.va.vro.service.provider.bgs.service;
 
-import static gov.va.vro.service.provider.camel.MasIntegrationRoutes.NEW_NOT_PRESUMPTIVE;
-import static gov.va.vro.service.provider.camel.MasIntegrationRoutes.PDF_UPLOAD_FAILED_AFTER_ORDER_EXAM;
-import static gov.va.vro.service.provider.camel.MasIntegrationRoutes.SUFFICIENCY_UNDETERMINED;
-
 import java.util.Map;
 
 public class BgsClaimNotes {
 
+  public static final String SUFFICIENCY_UNDETERMINED = "Sufficiency cannot be determined.";
+  public static final String PDF_UPLOAD_FAILED_AFTER_ORDER_EXAM = "docUploadFailed";
+  public static final String NEW_NOT_PRESUMPTIVE = "newClaimMissingFlash266";
```

Comment:
Should these be part of the `EventReason` enum? The concepts represented by these String constants seem to overlap with those in `EventReason`.

Reply:
Yes. I tried not to remove the code in this file before asking you to review it. I'll use EventReason to replace these constants.

Reply:
I replaced those constants with enum items.

Reply:
@yoomlam Can we merge this BgsClaimNotes with EventReason? One issue is those "key" (code in EventReason) used in BgsClaimNotes have different messages (narratives in EventReason).